### PR TITLE
Remove unnecessary scripts provided by boilerplate

### PR DESCRIPTION
**Pull request by AlexVulaj**
This change updates MNMO to use the standard boilerplate, bringing it in line with how other operator pipelines function.  This allows us to remove a lot of unnecessary scripts locally.

Also fixes a bug discovered after merging [this PR](https://github.com/openshift/managed-node-metadata-operator/pull/32).

The new files were generated/updated by running `make boilerplate-update` after initial changes were made.

### CODE REVIEW

File: int/int_test.go [hunk 1]
```diff
@@ -18,15 +18,14 @@ var (
...
-	MaxWaitTime time.Duration = 30 * time.Second
+	MaxWaitTime = 30 * time.Second
 )
 
 var _ = BeforeSuite(func() {
 	var err error
 	i, err = NewIntegration()
 	Expect(err).NotTo(HaveOccurred())
 	err = i.DisableWebhook()
-	Expect(err).NotTo(HaveOccurred())
```
Comment (iamkirkbater) [inline int/int_test.go:29]: Do we need to undo these changes to the integration tests since they were just symptoms and not the actual problems before we commit this?

Comment (AlexVulaj) [reply int/int_test.go:29]: I don't think we _need_ to, but we should.  I'll roll some of these back.
